tradeit-android-sdk/src/main/java/it/trade/android/sdk/TradeItConfigurationBuilder.java

```diff
@@ -32,6 +33,11 @@ public TradeItConfigurationBuilder withRequestInterceptor(RequestInterceptorParc
         return this;
     }
 
+    public TradeItConfigurationBuilder withStartFetchingBrokerList(boolean startFetchingBrokerList) {
```

Comment:
If by default it is set to true, we could just ad a method the set it to false, like `dontStartFetchingBrokerList` ?